### PR TITLE
Rework tables

### PR DESCRIPTION
**Pull request by JarbasAl**

### CODE REVIEW

File: docs/comparison.md [hunk 1]
```diff
@@ -1,42 +1,237 @@
...
+| fallback_tts            | ovos-tts-plugin-mimic                             | ovos-tts-plugin-mimic                             | ovos-tts-plugin-mimic                             |
+| **Recommended Plugins** |                                                   |                                                   |                                                   |
+| STT - On device         | ovos-stt-plugin-vosk                              | ovos-stt-plugin-vosk                              | ovos-stt-plugin-vosk                              |
+| STT - On premises       | ovos-stt-plugin-server<br>(any plugin)            | ovos-stt-plugin-server<br>(any plugin)            | ovos-stt-plugin-server<br>(any plugin)            |
+| STT - Cloud             | ovos-stt-plugin-server<br>(ovos proxy)            | ovos-stt-plugin-server<br>(ovos proxy)            | ovos-stt-plugin-server<br>(ovos proxy)            |
+| TTS - On device         | ovos-tts-plugin-mimic3                            | ovos-tts-plugin-mimic3                            | ovos-tts-plugin-mimic3                            |
```
Comment (builderjer) [inline docs/comparison.md:109]: Same as above about lack of power

File: docs/comparison.md [hunk 2]
```diff
@@ -1,42 +1,237 @@
...
+| Padatious Intents                  | Yes                                 | Yes                                 | Yes                                 |
+| OCP Skills                         | Yes                                 | Yes                                 | Partial<br>(audio only)             |
+| MNOS Common Query Skills           | Yes                                 | Yes                                 | Yes                                 |
+| MNOs Fallback Skills               | Yes                                 | Yes                                 | Yes                                 |
+| MNOs Skills                        | Yes                                 | Yes                                 | Yes                                 |
+| Mycroft Common Play Skills         | Partial<br>(imperfect compat layer) | Partial<br>(imperfect compat layer) | Partial<br>(imperfect compat layer) |
+| **Media Frameworks**               |                                     |                                     |                                     |
```
Comment (builderjer) [inline docs/comparison.md:171]: Most of the media frameworks here, I would like to include in the image.  Maybe add "WIP"

File: docs/comparison.md [hunk 3]
```diff
@@ -166,20 +284,77 @@ MycroftAI also provides some images, we do not recommend using any of these but
...
+| **Hardware - Compatibility** |
+| Raspberry Pi 3B              | **WIP**                           | No                   | Yes                                  |   
+| Raspberry Pi 3B+             | **WIP**                           | No                   | Yes                                  | 
+| Raspberry Pi 3A+             | **WIP**                           | No                   | Yes                                  |   
+| Raspberry Pi 4               | Yes                           | Yes                  | Yes                                  | 
+| X86_64                       | *planned*                     | No                   | No                                   |                     
+| Mark-1                       | **WIP**                       | No                   | No<br>*possibly in future*           |                        
+| Mark-2                       | **WIP**<br>(no leds)          | **WIP**<br>(no leds) | partial<br>(no gui + manual install) |           
+| Mark-2  (dev kit)            | Yes                           | Yes                  | partial<br>(no gui + manual install) |           
+| **Hardware - Peripherals**   |
```
Comment (builderjer) [inline docs/comparison.md:304]: These I would also like to support out of the box.  I am looking at how the buildroot image detects them and will try and add that to the code.  Maybe "WIP"

**Review by builderjer — COMMENTED** [review]
I have made a few comments concerning the raspbian image.  But all in all, I really like the way this is broken down.

File: docs/comparison.md (hunk 3 above)
Comment (JarbasAl) [reply docs/comparison.md:304]: i'd like to only use WIP if some functionality is there already, eg, HA plugin but no Voice interface yet

for the other i'd prefer to use "planned" if work didn't start yet

File: docs/comparison.md (hunk 1 above)
Comment (JarbasAl) [reply docs/comparison.md:109]: should test this one, but i think that mimic3 should run in a pi3 too

File: docs/comparison.md (hunk 3 above)
Comment (builderjer) [reply docs/comparison.md:304]: That works for me

**Pull request closed without merging**